inductiva/storage/storage.py

```diff
@@ -118,3 +129,154 @@ def _print_contents_table(contents):
         formatters=formatters,
         header_formatters=header_formatters,
     )
+
+
+def upload_from_url(
+    url: str,
+    remote_dir: str,
+    file_name: str = None,
+    unzip: bool = False,
+    overwrite: bool = True,
+):
+    """Upload a file from a URL to the user workspace."""
+    api_instance = storage_api.StorageApi(inductiva.api.get_client())
+
+    if file_name is None:
+        parsed_url = urlparse(url)
+        file_name = unquote(parsed_url.path.split("/")[-1])
+    contents = api_instance.upload_from_url(
+        query_params={
+            "url": url,
+            "file_name": file_name,
+            "unzip": "t" if unzip else "f",
+            "overwrite": "t" if overwrite else "f",
+        },
+        path_params={
+            "folder_name": remote_dir,
+        },
+    )
+    if contents.response.status == 200:
+        logging.info("File is being uploaded...")
+    else:
+        logging.info("File upload failed.")
+
+
+def upload(
```

Comment:
This method has a lot in common with the method to upload the input files to a task `upload_input`.
Lets refactor the code a bit to avoid all the code repetition.